### PR TITLE
[MULTIARCH-4227] IBM Z secure boot enablement

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.16+

Issue: 
https://issues.redhat.com/browse/MULTIARCH-4227

Link to docs preview:
- [z/VM](https://76571--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z.html#installation-user-infra-machines-iso-ibm-z_installing-ibm-z) 
    - [Verification](https://76571--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z.html#installation-complete-user-infra_installing-ibm-z) 
- [LPAR](https://76571--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z-lpar#installation-user-infra-machines-iso-ibm-z_installing-ibm-z-lpar)
    - [Verification](https://76571--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z-lpar#installation-complete-user-infra_installing-ibm-z-lpar) 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by alkleinrh** [comment]
LGTM

**Comment by nikita-dubrovskii** [comment]
LGTM!

**Comment by holgwolf** [comment]
LGTM

**Review by gwynnemonahan — COMMENTED** [review]
Hey @SNiemann15 ,

Just one comment as it looks like the <1> is missing from the description of what "coreos.live.rootfs_url=http://<http_server>/rhcos-<version>-live-rootfs.<architecture>.img \// <1>" means.

Thanks!

File: modules/installation-ibm-z-user-infra-machines-iso.adoc [hunk 1]
```diff
@@ -115,15 +122,15 @@ Additional postinstallation steps are required to fully enable multipathing. For
...
+The following is an example parameter file `worker-1.parm` for a compute node with multipathing:
 +
 [source,terminal]
 ----
 rd.neednet=1 \
 console=ttysclp0 \
 coreos.inst.install_dev=/dev/disk/by-id/scsi-<serial_number> \
-coreos.live.rootfs_url=http://cl1.provide.example.com:8080/assets/rhcos-live-rootfs.s390x.img \
-coreos.inst.ignition_url=http://cl1.provide.example.com:8080/ignition/worker.ign \
+coreos.live.rootfs_url=http://<http_server>/rhcos-<version>-live-rootfs.<architecture>.img \// <1>
```
[the remark this answers is not in the document]
Comment (SNiemann15) [reply modules/installation-ibm-z-user-infra-machines-iso.adoc:132]: Thanks for the finding 👍 That was a copy/paste typo from the above example. I don't think we need to call out the variable descriptions again. I'll update.

**Comment by gwynnemonahan** [comment]
LGTM

**Pull request merged**